components/aws/sagemaker/tests/integration_tests/component_tests/test_deploy_component.py

```diff
@@ -43,6 +43,9 @@ def np2csv(arr):
 @pytest.mark.parametrize(
     "test_file_dir",
     [
+        pytest.param(
+            "resources/config/kmeans-mnist-deploy", marks=pytest.mark.canary_test
```

Comment:
Should this be canary? We already have a canary test for deploy.

Reply:
Also can we rename the test to reflect update-endpoint in the name

Reply:
Both comments addressed